### PR TITLE
Rahul/send names

### PR DESCRIPTION
**Pull request by rahul-ramesh**

### CODE REVIEW

File: protocol/protocol.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func (p Protocol) ReadFunctionPublicKey() (*entities.FunctionPublicKey, error) {
 	return readMsg[entities.FunctionPublicKey](p.Websocket)
 }
 
-func (p Protocol) WriteFunctionPublicKey(key string) error {
-	return writeMsg(p.Websocket, entities.FunctionPublicKey{FunctionTokenPublicKey: key})
+func (p Protocol) WriteFunctionPublicKey(key string, name string) error {
```
Comment (justin1121) [inline protocol/protocol.go:93]: think it was mentioned in the related PR in enclave-supervisor but this should be renamed to something more generic.

**Review by justin1121 — CHANGES_REQUESTED** [review]
Just naming things.

File: protocol/protocol.go (hunk 1 above)
Comment (rahul-ramesh) [reply protocol/protocol.go:93]: Changed the name!

**Review by justin1121 — APPROVED** [review]
LGTM

**Pull request merged**